README.md

```diff
@@ -29,6 +29,35 @@ For more details see [CocoaPods website](http://cocoapods.org)
 Add `github "lukkas/Visually"` to your Cartfile, run `carthage update` and drag built frameworks to your Xcode project.
 For more details see [Carthage git repository](https://github.com/Carthage/Carthage)
 
+### Swift Package Manager
+
+To install Visually using **Swift Package Manager** go through following steps:
+
+1. Add following package dependency in you **Package.swift** ``` .package(url: "https://github.com/lukkas/Visually.git", from: "2.0.0") ```
```

Comment:
I think I'd just leave point 1. (and of course remove the number) and maybe add some context here, like so
```swift
dependencies: [
         .package(url: "https://github.com/lukkas/Visually.git", from: "2.0.0")
]
```
I'd leave out point 2 and the example. It is just information that you can find anywhere and I wouldn't like to focus on this in README too much

Reply:
The "from" must be specified for a package dependency, I suggest changing "2.0.0" to -> "VERSION_NUMBER_HERE"
what do you think?

Reply:
I meant the entire second point of the instruction, not the version number. I don't want to get too specific of how to use SPM here. I think that up to 5 lines, including snippets is enough.

Reply:
Sure, I'm onto it :)